src/types.d.ts

```diff
@@ -23,19 +22,6 @@ export type Visitors<T extends BaseNode = BaseNode> = T['type'] extends '_'
 
 export { Context };
 
-export type TypeAnnotationNodes =
-	| TSESTree.TypeNode
-	| TSESTree.TypeElement
-	| TSESTree.TSTypeAnnotation
-	| TSESTree.TSPropertySignature
-	| TSESTree.TSTypeParameter
-	| TSESTree.TSTypeParameterDeclaration
-	| TSESTree.TSTypeParameterInstantiation
-	| TSESTree.TSEnumMember
-	| TSESTree.TSInterfaceHeritage
-	| TSESTree.TSClassImplements
-	| TSExpressionWithTypeArguments;
```

Comment:
I removed this as it's not used anymore anywhere... And since `TSESTree` was not a deps, no one could use it outside? 
Safe deleting it?